phpBB/styles/prosilver/theme/colours.css

```diff
@@ -116,7 +116,7 @@ a:hover { color: #D31141; }
 	color: #000000;
 }
 
-.post h3 a.first-unread { {
+.post h3 a.first-unread {
```

Comment:
I am not sure if the usual old behaviour can be compromised with this change, I mean while you are into the topic and click onto the post's title (target) it should change to black. perhaps it should read like - not tested.

```
.post h3 a.first-unread,
.post:target h3 a  {
```